### ISSUE TITLE
Worker timeout every 17min

### ISSUE THREAD

**Issue by jpxd** [mormahr/pdf-service#124]
When leaving the pdf-service container running for some time without any load, every 17mins the workers quit with a critical error and are respawned.

Example log:
```
weasyprint_1  | [2021-08-29 12:42:14 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:9)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:10)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [9] [INFO] Worker exiting (pid: 9)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:11)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [10] [INFO] Worker exiting (pid: 10)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [11] [INFO] Worker exiting (pid: 11)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:12)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [12] [INFO] Worker exiting (pid: 12)
weasyprint_1  | [2021-08-29 12:42:14 +0000] [169] [INFO] Booting worker with pid: 169
weasyprint_1  | [2021-08-29 12:42:14 +0000] [170] [INFO] Booting worker with pid: 170
weasyprint_1  | [2021-08-29 12:42:14 +0000] [171] [INFO] Booting worker with pid: 171
weasyprint_1  | [2021-08-29 12:42:15 +0000] [172] [INFO] Booting worker with pid: 172
weasyprint_1  | [2021-08-29 12:59:36 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:169)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:170)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [169] [INFO] Worker exiting (pid: 169)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:171)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:172)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [171] [INFO] Worker exiting (pid: 171)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [170] [INFO] Worker exiting (pid: 170)
weasyprint_1  | [2021-08-29 12:59:36 +0000] [172] [INFO] Worker exiting (pid: 172)
weasyprint_1  | [2021-08-29 12:59:37 +0000] [329] [INFO] Booting worker with pid: 329
weasyprint_1  | [2021-08-29 12:59:37 +0000] [8] [WARNING] Worker with pid 171 was terminated due to signal 9
weasyprint_1  | [2021-08-29 12:59:37 +0000] [330] [INFO] Booting worker with pid: 330
weasyprint_1  | [2021-08-29 12:59:37 +0000] [331] [INFO] Booting worker with pid: 331
weasyprint_1  | [2021-08-29 12:59:37 +0000] [332] [INFO] Booting worker with pid: 332
weasyprint_1  | [2021-08-29 13:17:56 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:329)
weasyprint_1  | [2021-08-29 13:17:56 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:330)
weasyprint_1  | [2021-08-29 13:17:56 +0000] [8] [CRITICAL] WORKER TIMEOUT (pid:331)
```

**Comment by jpxd** [comment]
Relevant Gunicorn documentation: https://docs.gunicorn.org/en/latest/settings.html#timeout
Setting the timeout to 0 should work as we want the workers to run forever, right?

**Comment by mormahr** [comment]
Solved with v1.1.0 release.
